.github/workflows/docs.yml

```
@@ -112,3 +125,4 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deploy
         uses: actions/deploy-pages@v5
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)